SigningBoxInput/SigningBoxInput.abi.json

```diff
@@ -0,0 +1,22 @@
+{
+	"ABI version": 2,
+	"header": ["time"],
+	"functions": [
+		{
+			"name": "get",
+			"inputs": [
+				{"name":"answerId","type":"uint32"},
+				{"name":"prompt","type":"bytes"},
+				{"name":"possiblePublicKeys","type":"uint256[]"}
+			],
+			"outputs": [
+				{"name":"handle","type":"uint32"},
+				{"name":"pubkey","type":"uint256"}
```

Comment:
I've just realized that we might cannot return the `pubkey` to Debot with the best UX (at least, for security card as it would require to "touch" the phone).

I would suggest to return only the Signing Box handle, and get the pubkey via SDK directly knowing the handle once its needed

Reply:
Ok. Let's return only handle.